### PR TITLE
KOGITO-5410 Disable Failing Native Image tests on kogito-apps

### PR DESCRIPTION
**Pull request by evacchi**

### CODE REVIEW

**Comment by evacchi** [comment]
jenkins run native

**Review by tarilabs — COMMENTED** [review]
I'm not sure this are "randomic failures" hence if this is a good idea to disable it, as it might be a symptom some NI generation is actually breaking?

**Comment by danielezonca** [comment]
I agree that it looks strange to disable all of them, @evacchi is it a regression introduced by quarkus 2.0 update?

**Comment by evacchi** [comment]
@danielezonca @tarilabs it's been broken on master for a while; I only discovered it recently as I was running the native build on this PR https://github.com/kiegroup/kogito-runtimes/pull/1277 and it seemed an unrelated failure. I am disabling them because as they are they are only masking other possible issues. We can disable the native build entirely if that is what you mean. I have created tickets for tracking the issue in follow ups.

I can only assume it's been broken for a while https://eng-jenkins-csb-business-automation.apps.ocp4.prod.psi.redhat.com/job/KIE/job/kogito/job/nightly/job/master/job/kogito-native/

unfortunately there were other broken native tests on runtimes that probably masked it on apps see https://github.com/kiegroup/kogito-runtimes/pull/1379

it is probably unrelated to Quarkus 2.0

**Comment by tarilabs** [comment]
This https://github.com/kiegroup/kogito-apps/pull/907 should fix Native test for the JIT Runner.
Thanks for reporting.

File: jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeJITDMNResourceIT.java [hunk 1]
```diff
@@ -16,10 +16,12 @@
 
 package org.kie.kogito.jitexecutor.runner;
 
+import org.junit.jupiter.api.Disabled;
 import org.kie.kogito.jitexecutor.dmn.api.JITDMNResourceTest;
 
 import io.quarkus.test.junit.NativeImageTest;
 
+@Disabled("KOGITO-5409 JIT Executor Native Image tests fail")
```
Comment (tarilabs) [inline jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeJITDMNResourceIT.java]:
```suggestion
```

File: jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeOneOfEachTypeIT.java [hunk 2]
```diff
@@ -16,10 +16,12 @@
 
 package org.kie.kogito.jitexecutor.runner;
 
+import org.junit.jupiter.api.Disabled;
 import org.kie.kogito.jitexecutor.dmn.OneOfEachTypeTest;
 
 import io.quarkus.test.junit.NativeImageTest;
 
+@Disabled("KOGITO-5409 JIT Executor Native Image tests fail")
```
Comment (tarilabs) [inline jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeOneOfEachTypeIT.java]:
```suggestion
```

File: jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeSchemaResourceIT.java [hunk 3]
```diff
@@ -16,10 +16,12 @@
 
 package org.kie.kogito.jitexecutor.runner;
 
+import org.junit.jupiter.api.Disabled;
 import org.kie.kogito.jitexecutor.dmn.api.SchemaResourceTest;
 
 import io.quarkus.test.junit.NativeImageTest;
 
+@Disabled("KOGITO-5409 JIT Executor Native Image tests fail")
```
Comment (tarilabs) [inline jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeSchemaResourceIT.java]:
```suggestion
```

File: jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeValidatorResourceIT.java [hunk 4]
```diff
@@ -16,11 +16,13 @@
 
 package org.kie.kogito.jitexecutor.runner;
 
+import org.junit.jupiter.api.Disabled;
 import org.kie.kogito.jitexecutor.dmn.api.ValidatorResourceTest;
 
 import io.quarkus.test.junit.NativeImageTest;
 
+@Disabled("KOGITO-5409 JIT Executor Native Image tests fail")
```
Comment (tarilabs) [inline jitexecutor/jitexecutor-runner/src/test/java/org/kie/kogito/jitexecutor/runner/NativeValidatorResourceIT.java]:
```suggestion
```

**Review by tarilabs — CHANGES_REQUESTED** [review]
ref https://github.com/kiegroup/kogito-apps/commit/8f0f6e7eeb7e52025b47e5074dbb2e45731973db

**Comment by evacchi** [comment]
@tiagodolphine this disables the test -- if that's ok I can merge it, unless you want to take a look already

**Review by tiagodolphine — APPROVED** [review]
It's fine for me, TBF I'm not even sure we need native build for job-service, if that's the case we could remove the native tests.

**Comment by evacchi** [comment]
jenkins run native

**Comment by evacchi** [comment]
Jenkins retest this

**Comment by evacchi** [comment]
Jenkins retest this

**Comment by evacchi** [comment]
jenkins run native

**Comment by evacchi** [comment]
jenkins run native

**Comment by evacchi** [comment]
Jenkins retest this

**Comment by evacchi** [comment]
there are some further native tests breaking now, needs further investigation?

**Pull request closed without merging**